### ISSUE TITLE
Add discourse features from Levinsohn?

### ISSUE THREAD

**Issue by jonathanrobie** [biblicalhumanities/greek-new-testament#22]
Should we add some discourse features from Levinsohn to the Lowfat treebanks?

See http://www.ibiblio.org/bgreek/forum/viewtopic.php?f=11&t=4049&p=27238#p27238.

**Comment by jcuenod** [comment]
I would vote yes :)

**Comment by jtauber** [comment]
How far do we want to go having a single tree that contains all analyses of interest?

Even if we ship files that have everything combined, it might be better to make those files generated from the sources where possible. Otherwise they'll become quickly out of date if updates are made upstream.

**Comment by jonathanrobie** [comment]
Yes, I think we should distinguish shipping combined files from maintaining the sources.

**Comment by jcuenod** [comment]
That's the strength of text-fabric's approach to the data

**Comment by jonathanrobie** [comment]
The truth is often in the middle.  The relationships among data and the way they are updated are both important for choosing how to maintain the data.

**Comment by emg** [comment]
I agree, Jonathan.

I would add that one of Emdros's strengths is that it can contain all of the data, even in the face of overlapping and/or orthogonal hierarchies. It can then expose different "views" of that data, while keeping everything searchable.

**Comment by jonathanrobie** [comment]
Text Fabric and Emdros are two great systems that can contain data and expose it in various ways.  This can even be done at the raw data level with an XML database, and you can imagine any number of other systems that could do the same.

Back to this issue: In its fullest form, Levinsohn's data uses a different hierarchy than the syntax hierarchy, points of departure do not always align with sentence boundaries.  That means we need two separate hierarchies.  I don't know whether there is value to putting some of his analysis in the syntax trees or not.
